### DISCUSSION TITLE
Breakdown by domain name for external link clicks

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#1229] [Anything Else]
"it would be great if you'd implement a breakdown by domain so we can see the total amount of clicks that went to site A and site B, etc."

**Update:** It’s now possible to group external link clicks by the domain name or any other keyword

**Comment by SemaphoreOxalis** [comment]
I add my idea here to achieve this :  add wildcard filtering on external link 

- filter on the outbound link goal
- click on a goal URL
- edit filter setting : Edit filter is property
- group URLs using ** wildcard : value is [www.someurl.com/](http://www.someurl.com/)**

![218523537-3f680198-62cc-40ee-bd71-68946a91e83d](https://user-images.githubusercontent.com/10586254/218569793-40763da2-b425-4fc9-adc0-68d0ee8646b5.png)

with this we can get the total visitors, devices source on a group of external link

**Comment by metmarkosaric** [comment]
**Update:** It’s now possible to group external link clicks by the domain name or any other keyword.

In the “Filter” button in the top right of the dashboard, choose “Property”. Then select “url” (that includes all the external link click urls we have recorded) and use the “contains” option. Add whatever domain or keyword you want to group the external link clicks by

<img width="559" alt="Screenshot 2024-02-20 at 14 49 05" src="https://github.com/plausible/analytics/assets/34340819/85c14131-15b2-4756-9283-3e940757cae8">

**Reply by SemaphoreOxalis** [reply]
Just to be clear for everyone : we need to select the good goal too because URL property could be found in multiple event/goal ;-)

**Reply by metmarkosaric** [reply]
that's right @SemaphoreOxalis, thanks for the note! if you want to narrow the results, add the goal itself to the filter. btw you can group error pages or file downloads (or any other property you track) this way too
